catalog/video/video-admin-plugin/pom.xml

```diff
@@ -90,7 +90,7 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.60</minimum>
+                                            <minimum>0.55</minimum>
```

Comment:
⭐️  Can we add unit tests for the new `networkInterfaces` method and get those numbers back up?